### DISCUSSION TITLE
Statusline transparency for a character

### DISCUSSION THREAD

**Discussion by ChrisAmelia** [neovim/neovim#31114] [Q&A] [answered]
Following this commit https://github.com/neovim/neovim/commit/e049c6e4c08a141c94218672e770f86f91c27a11
I have troubles updating my config to keep the same statusline: keeping the transparency for the character , I could set it with `guibg=none`.

**Before**
![image](https://github.com/user-attachments/assets/1d2d3d74-4b74-4bc4-a77d-65c56edab5b5)
```lua
bvim.api.nvim_command("hi HIGHLIGHT_GIT guifg=#0066FF guibg=none")
statusline = statusline .. "%#HIGHLIGHT_GIT#"
statusline = statusline .. ""
```

**After (attempt)**
![image](https://github.com/user-attachments/assets/09f7d0c9-d606-410d-83d3-a7d032d1b392)
```lua
vim.api.nvim_command("hi HIGHLIGHT_GIT guibg=#0066FF")
statusline = statusline .. "%#HIGHLIGHT_GIT#"
statusline = statusline .. ""
```

**Comment by zeertzjq** [comment]
What is the output of `:hi StatusLine`?

**Reply by ChrisAmelia** [reply]
```
:hi StatusLine                                                                                                                                                                                                                          
StatusLine     xxx cterm=reverse ctermfg=59 ctermbg=234 gui=reverse guifg=#75715e guibg=#272822
```

**Reply by zeertzjq** [reply, marked as answer]
You'll need both `guibg=NONE` and `gui=nocombine` for `HIGHLIGHT_GIT`.

**Reply by ChrisAmelia** [reply]
Thanks, works like a charm.
